TCL/polarDensity_for_DTA.tcl

```diff
@@ -486,7 +494,7 @@ proc loop_over_frames {shell species headname tailname lipidbeads_selstr start_f
 proc loop_over_shells {species headname tailname lipidbeads_selstr low_f upp_f low_f_avg upp_f_avg} {
     global params
     set delta_frame [expr ($params(end_frame) - $params(start_frame)) / $params(dt)]
-    for {set ri $params(Rmin)} { $ri<=$params(Rmax)} { set ri [expr $ri + $params(dr)]} {
```

Comment:
The equals sign meant that when Rmax was evenly divisible by dr, analysis would exceed Rmax by one whole shell